dags/openshift_nightlies/config/install/rosa/rosa-hcp-ovn.json

```diff
@@ -0,0 +1,23 @@
+{
+    "rosa_hcp": "true",
+    "aws_profile": "",
+    "aws_access_key_id": "",
+    "aws_secret_access_key": "",
+    "aws_authentication_method": "sts",
+    "aws_region": "us-east-2",
+    "rosa_environment": "staging",
+    "rosa_cli_version": "master",
+    "ocm_environment": "stage",
+    "managed_channel_group": "nightly",
+    "managed_ocp_version": "latest",
+    "openshift_worker_count": 7,
+    "openshift_network_type": "OVNKubernetes",
+    "openshift_worker_instance_type": "m5.2xlarge",
+    "machineset_metadata_label_prefix": "machine.openshift.io",
+    "staging_mgmt_cluster_name": "hs-mc-mdqo7k4u0-pjj8f", 
+    "staging_mgmt_provisioner_shards": "0085c398-726e-11ed-a42d-0a580a810706",
+    "public_subnet": "subnet-049e1ebe7f078f4b8",
+    "private_subnet": "subnet-0e6cea56f04ac9854",
```

Comment:
This is something that it's going to exist forever?
A permanent mgmt cluster?

Reply:
Yes for next 3-4 months, so do subnets.

Reply:
this is replaced with new VPC creation for every cluster.